docs/program-setup/serial/serial_communication.md

```diff
@@ -2,7 +2,7 @@
 
 The Arm Mbed microcontroller on your board can communicate with a host PC over the same USB cable that you use for programming.
 
-<span class="tips">If you're working on Windows earlier than Windows 10, you might need to [install a serial driver](../program-setup/windows-serial-driver.html).</span>
+<span class="tips">If you're working on Windows 7, you might need to [install a serial driver](../program-setup/windows-serial-driver.html).</span>
```

Comment:
So what about Windows 8? Does that work?

Reply:
Yes, it works on Windows 8.

For example, https://github.com/ARMmbed/mbed-cli-windows-installer/blob/b76911430dffbb6e6e751e98a19ac4e160a7c3b4/source/install.nsi#L238-L243

Reply:
To clarify, it shouldn't be use don Windows 8 either

Reply:
Sorry by "works" I meant no driver needed